Add missing FTL config options (#564)
pi-hole/docs: docs/ftldns/configfile.md

Despite `REFUSE` sounding similar to `DROP`, it turned out that many clients will just immediately retry, causing up to several thousands of queries per second. This does not happen in `DROP` mode.

#### `MOZILLA_CANARY=true|false` (PR [#1148](https://github.com/pi-hole/FTL/pull/1148)) {#mozilla_canary data-toc-label='Mozilla canary domain handling'}

> Should Pi-hole always replies with `NXDOMAIN` to `A` and `AAAA` queries of `use-application-dns.net` to disable Firefox automatic DoH?

Reviewer: ```suggestion
Should Pi-hole always replies with `NXDOMAIN` to `A` and `AAAA` queries of `use-application-dns.net` to disable Firefox automatic DNS-over-HTTP?
```

Should `FTL` try to resolve IPv4 addresses to hostnames?

#### `PIHOLE_PTR=true|false` (PR [#1111](https://github.com/pi-hole/FTL/pull/1111)) {#pihole_ptr data-toc-label='Pi-hole PTR'}

> FTL will reply with `pi.hole` for every address for which a local interface exists. When you set this option to false, Pi-hole will not add any PTR requests itself.

Reviewer: ```suggestion
FTL will reply with `pi.hole` for every address for which a local interface exists. When you set this option to `false`, Pi-hole will respond with `pi.hole` on PTR requests to local interface addresses.
```

Author: false= **not** responsed?

Reviewer: Yes.

**[More details](../regex/overview.md)**

> #### `DEBUG_API=false|true` {#debug_api data-toc-label='Telnet'}

Reviewer: ```suggestion
#### `DEBUG_API=false|true` {#debug_api data-toc-label='API'}
```

Author: I removed `API` on purpose - with this auto-hint thing it completely destroys the TOC entry and the heading. Have a look at the current version: https://docs.pi-hole.net/ftldns/configfile/#debug_apifalsetrue-debug_api-data-toc-labeltelnet-api

Reviewer: Okay, so we need to change this for v6.0 but this is okay as a workaround for now.

Log information about script helpers, e.g., due to `dhcp-script`.

#### `ADDR2LINE=true|false` (PR [#774](https://github.com/pi-hole/FTL/pull/774)) {#addr2line data-toc-label='Addr2Line'}

> Should FTL translate its own stack addresses into code lines during the bug backtrace? This improves the analysis of crashed significantly and it is recommended to leave the option enabled.

Reviewer: ```suggestion
Should FTL translate its own stack addresses into code lines during the bug backtrace? This improves the analysis of crashed significantly. It is recommended to leave the option enabled. This option should only be disabled when `addr2line` is known to not be working correctly on the machine because, in this case, the malfunctioning `addr2line` can prevent from generating any backtrace at all.
```
